src/wazuh_db/schema_global_upgrade_v3.sql

```diff
@@ -0,0 +1,48 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ *
+ * October, 2020.
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+CREATE TABLE IF NOT EXISTS _agent (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL,
+    ip TEXT,
+    register_ip TEXT,
+    internal_key TEXT,
+    os_name TEXT,
+    os_version TEXT,
+    os_major TEXT,
+    os_minor TEXT,
+    os_codename TEXT,
+    os_build TEXT,
+    os_platform TEXT,
+    os_uname TEXT,
+    os_arch TEXT,
+    version TEXT,
+    config_sum TEXT,
+    merged_sum TEXT,
+    manager_host TEXT,
+    node_name TEXT DEFAULT 'unknown',
+    date_add INTEGER NOT NULL,
+    last_keepalive INTEGER,
+    `group` TEXT DEFAULT 'default',
+    sync_status TEXT NOT NULL CHECK (sync_status IN ('synced', 'syncreq')) DEFAULT 'synced',
+    connection_status TEXT NOT NULL CHECK (connection_status IN ('pending', 'never_connected', 'active', 'disconnected')) DEFAULT 'never_connected',
+    disconnected_time INTEGER DEFAULT 0
+);
+
+BEGIN;
+
+INSERT INTO _agent (id, name, ip, register_ip, internal_key, os_name, os_version, os_major, os_minor, os_codename, os_build, os_platform, os_uname, os_arch, version, config_sum, merged_sum, manager_host, node_name, date_add, last_keepalive, `group`, sync_status, connection_status) SELECT id, name, ip, register_ip, internal_key, os_name, os_version, os_major, os_minor, os_codename, os_build, os_platform, os_uname, os_arch, version, config_sum, merged_sum, manager_host, node_name, date_add, last_keepalive, `group`, sync_status, connection_status FROM agent;
+UPDATE _agent SET disconnected_time = CASE WHEN id != 0 AND connection_status = 'disconnected' AND disconnected_time = 0 THEN STRFTIME('%s', 'NOW') ELSE disconnected_time END;
+UPDATE metadata SET value = '3' where key = 'db_version';
+
+END;
+
+DROP TABLE IF EXISTS agent;
+ALTER TABLE _agent RENAME TO agent;
```

Comment:
Please, add a new line at the end of the file

Reply:
Done